Patches: Fake players injuring underlings crashes server & librarians in overworld getting dimension specific trades crash client on server (#513)
mraof/Minestuck: CHANGELOG.md

## [Unreleased]

### Fixed

> - Fake players no longer crash the game when injuring underlings

Reviewer: ```suggestion
- Fix a crash from fake players injuring underlings
```
